### PR TITLE
add python solution for Why Did the Cow Cross the Road II (715)

### PR DESCRIPTION
**Pull request by jjliewie**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/usaco-715.mdx [hunk 1]
```diff
@@ -98,4 +98,38 @@ public class MaxCross {
...
+
+<PySection>
+
+```python
+import sys
+
+sys.stdin = open("maxcross.in", "r")
+sys.stdout = open("maxcross.out", "w")
+
+input = sys.stdin.readline
```
Comment (ryanchou-dev) [inline solutions/usaco-715.mdx:110]: .

File: solutions/usaco-715.mdx [hunk 2]
```diff
@@ -98,4 +98,38 @@ public class MaxCross {
...
+for _ in range(b):
+	id[int(input())] = 1
+
+for i in range(left, right + 1):
+	tmp += id[i]
+
+possible = [tmp]
+
+while n > right:
+
```
Comment (ryanchou-dev) [inline solutions/usaco-715.mdx:125]:
```suggestion
```

File: solutions/usaco-715.mdx (hunk 1 above)
Comment (jjliewie) [reply solutions/usaco-715.mdx:110]: okay

File: solutions/usaco-715.mdx [hunk 3]
```diff
@@ -98,4 +98,30 @@ public class MaxCross {
 ```
 
 </JavaSection>
+
+<PySection>
+
+```python
+with open('maxcross.in') as read:
+	n, k, b = map(int, read.readline().split())
+	id, left, right, tmp = [0] * (n + 1), 1, k, 0
```
Comment (SansPapyrus683) [inline solutions/usaco-715.mdx:107]: don't use `id` as a var name since it overrides some builtin function

File: solutions/usaco-715.mdx [hunk 4]
```diff
@@ -98,4 +98,30 @@ public class MaxCross {
...
+	
+	for _ in range(b):
+		id[int(read.readline())] = 1
+
+for i in range(left, right + 1):
+	tmp += id[i]
```
Comment (SansPapyrus683) [inline solutions/usaco-715.mdx:113]: tmp is a bad name, you literally use it throughout the entire program

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/usaco-715.mdx (hunk 3 above)
Comment (jjliewie) [reply solutions/usaco-715.mdx:107]: okay

File: solutions/usaco-715.mdx (hunk 4 above)
Comment (jjliewie) [reply solutions/usaco-715.mdx]: okay

**Review by SansPapyrus683 — APPROVED** [review]

File: solutions/usaco-715.mdx [hunk 5]
```diff
@@ -2,7 +2,7 @@
 id: usaco-715
 source: USACO Silver 2017 February
 title: Why Did the Cow Cross the Road II
-author: Albert Zhu
+author: Albert Zhu, Juheon Rhee
 ---
 [Official Analysis](http://www.usaco.org/current/data/sol_maxcross_silver_feb17.html)
```
Comment (ryanchou-dev) [inline solutions/usaco-715.mdx:7]:
```suggestion
[Official Analysis (C++)](http://www.usaco.org/current/data/sol_maxcross_silver_feb17.html)
```

**Review by ryanchou-dev — APPROVED** [review]
⭐

**Pull request merged**